common/docsV1.md

```diff
@@ -1059,7 +1059,7 @@ The Patient endpoint also supports a GET /Patient operation where you can suppl
         },
         "identifier": [
           {
-            "system": "http://hl7.org/fhir/sid/us-mbi",
+            "system": "http://terminology.hl7.org/NamingSystem/cmsMBI",
```

Comment:
This isn't actually a link. Do you know if changing the attestation to this system works?

Reply:
it seemed like this was the up-to-date link for the identifier system, but I'm not entirely sure

Reply:
Linking to the page, the URI is still: 
http://hl7.org/fhir/sid/us-mbi

Reply:
[http://terminology.hl7.org/NamingSystem/cmsMBI](http://terminology.hl7.org/NamingSystem/cmsMBI) is the web link to where you can find the definition of CmsMBI.  http://hl7.org/fhir/sid/us-mbi is the actual URI used as the Identifier System.